modules/ossm-about-service-mesh-custom-resource-definitions.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// * install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-service-mesh-custom-resource-definitions_{context}"]
+= About Service Mesh Custom Resource Definitions
+
+Installing the {SMProductName} Operator also installs Custom Resource Definitions (CRD) that network administrators can use to configure {istio} for {SMProductShortName} installations. There are two different categories of CRDs: {SMProduct} CRDs and {istio} CRDs.
```

Comment:
Is there a reason we said "network administrators" here rather than just "administrators"? Our users are broader than network admins, so leaving the network park out seems more accurate.

Reply:
@longmuir I used that term because of internal norms and parlance.  Way back, we used to describe the customer as a developer. Then, it changed to more of a network admin that configures the mesh, and one or more developers use the mesh.  

I changed it to administrator here and can be mindful of the change going forward. If you'd like me to change the term back, feel free to comment here or message me in slack. Thanks!